README.md

```diff
@@ -105,8 +105,25 @@ Yamashita | Yah-MAH-sh-Ta  | some vowels (i, u) get removed (called devoicing),
 Kyoko     | Kyoh-koh       | (not Kee-oh-koh)      
 Ryousuke  | Ryoh-skeh      | the -suke suffix is usually skeh
 
+## Portuguese (Brazilian)
+
+The challenge with Portuguese is that there are some phonems (such as \[ɐ̃w̃\], \[s̻\] or \[ʎ\]) and [diacritics](https://en.wikipedia.org/wiki/Diacritic) (such as an accent or cedilla) not common or not at all present in the English language. 
```

Comment:
i'm trying to keep linguistics terminology to a minimum as I want this to be a guide for normies, not language nerds like me.  Can we just call them sounds instead of phonemes and I'm trying to avoid IPA where possible?   I think diacritics is ok because you immediately give an example.